### PR TITLE
cleanup: remove old landing page

### PR DESCRIPTION
**Pull request by MaximilianMeister**
for the upgrade from cloud 5 to 6 it was necessary to have a landing
page to choose to continue from cloud 5 upgrade or to perform a
fresh installation.
this is no longer needed for cloud 7 and we can immediately go to
the installer.

### CODE REVIEW

File: crowbar_framework/config/routes.rb [hunk 1]
```diff
@@ -194,7 +194,7 @@
   end
 
   scope :installer do
-    root to: "installer#index",
+    root to: "installer/installers#show",
```
Comment (jsuchome) [inline crowbar_framework/config/routes.rb]: This is already implemented somewhere?

Comment (MaximilianMeister) [inline crowbar_framework/config/routes.rb]: yes, that's the installer page

**Comment by rsalevsky** [comment]
Can we also drop the old upgrade related parts? See https://github.com/crowbar/crowbar-core/pull/667

**Comment by vuntz** [comment]
> Can we also drop the old upgrade related parts? See #667

Yes, we should. But it doesn't need to be in this PR :-)

**Pull request merged**